tests/SingleDatabaseTenancyTest.php

```diff
@@ -61,7 +61,7 @@
     expect(Comment::count())->toBe(2);
 });
 
-test('secondary models a r e scoped to the current tenant when accessed directly and parent relationship traitis used', function () {
+test('secondary models ARE scoped to the current tenant when accessed directly and parent relationship traits used', function () {
```

Comment:
`traitis` perhaps should be `trait is`, rather than `trait`? @abrardev99

Reply:
Right, that makes more sense

Reply:
Yeah it is `"trait is"`.

Reply:
Fixed it